pkgs/tools/X11/sunpaper/default.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, sunwait
+, wallutils
+, rPackages
+, pkgs
+}:
+
+stdenv.mkDerivation rec {
```

Don't think you need a C compiler for this (^:
```suggestion
stdenvNoCC.mkDerivation rec {
```

---

To use `stdenvNoCC` you'd have to change the input to the function above as well where it says `stdenv` :slightly_smiling_face:

Otherwise it's not in scope.